core/server/fluxruntime.go

```diff
@@ -109,37 +109,63 @@ func (cs *coreServer) ListFluxCrds(ctx context.Context, msg *pb.ListFluxCrdsRequ
 		return nil, fmt.Errorf("error getting impersonating client: %w", err)
 	}
 
-	var results []*pb.Crd
+	clist := clustersmngr.NewClusteredList(func() client.ObjectList {
+		return &apiextensions.CustomResourceDefinitionList{}
+	})
 
 	respErrors := []*pb.ListError{}
 
+	if err := clustersClient.ClusteredList(ctx, clist, false); err != nil {
+		var errs clustersmngr.ClusteredListError
+
+		if !errors.As(err, &errs) {
+			return nil, fmt.Errorf("CRDs clustered list: %w", errs)
+		}
+
+		for _, e := range errs.Errors {
+			respErrors = append(respErrors, &pb.ListError{
+				ClusterName: e.Cluster,
+				Message:     e.Err.Error(),
+			})
+		}
+	}
+
+	results := []*pb.Crd{}
+
 	opts := client.MatchingLabels{
 		coretypes.PartOfLabel: FluxNamespacePartOf,
 	}
 
-	list := &apiextensions.CustomResourceDefinitionList{}
+	for clusterName, lists := range clist.Lists() {
+		for _, l := range lists {
+			list, ok := l.(*apiextensions.CustomResourceDefinitionList)
+			if !ok {
+				continue
```

Comment:
Are we sure we want to just `continue` here? Has something gone horribly wrong if we fail this type assertion? (not rhetorical, I really don't know).

Reply:
Good question.  I saw that is what was done for the tf stuff so i just mirrored it.  IMO continue seems ok.  We could add some logging so something but we probably don't want to throw an error back to the user